.github/workflows/deploy.yml

```diff
@@ -72,6 +74,16 @@ jobs:
           API_KEY: ${{ secrets.API_KEY }}
         run: ibmcloud login -a "https://cloud.ibm.com" -u apikey -p "$API_KEY" -o "carbon-design-system" -s "production" -r "us-south"
 
-      - name: Deploy website
+      - name: Deploy to IBM Cloud
         run: |
           ibmcloud cf v3-zdt-push carbon-website -b https://github.com/cloudfoundry/nginx-buildpack.git
+
+      - name: Deploy to Vercel
```

Comment:
This let's us continue using the Vercel production deployment for our feedback dialog serverless function